docs/sources/tutorials/send-logs-to-loki.md

```diff
@@ -1,26 +1,28 @@
 ---
-canonical: https://grafana.com/docs/alloy/latest/tutorials/get-started/
-description: Getting started with Alloy
-title: Get started with Alloy
-weight: 10
+canonical: https://grafana.com/docs/alloy/latest/tutorials/send-logs-to-loki/
+aliases:
+  - ./get-started/ #/docs/alloy/latest/tutorials/get-started/
+description: Learn how to use Grafana Alloy to send logs to Loki
+menuTitle: Send logs to Loki
+title: Use Grafana Alloy to send logs to Loki
+weight: 100
 ---
 
-## Get started with {{% param "PRODUCT_NAME" %}}
+## Use {{% param "FULL_PRODUCT_NAME" %}} to send logs to Loki
 
-This tutorial shows you how to configure {{< param "PRODUCT_NAME" >}} to collect logs from your local machine, filter non-essential log lines, and send them to Loki, running in a local Grafana stack.
+This tutorial shows you how to configure {{< param "PRODUCT_NAME" >}} to collect logs from your local machine, filter non-essential log lines, send them to Loki, and use a Grafana dashboard to query and visualize the results.
```

Comment:
I don't think we use a Grafana dashboard during these tutorials, rather the Explore feature. Not sure if we need to be that precise, but wanted to point it out.

Reply:
Good point. I'll update the wording here to say "explore the results'.